oracle/service.go

```diff
@@ -381,15 +459,23 @@ func (s *oracleSvc) commitSetPrices(dataC <-chan *PriceData) {
 				submitBatch(prevBatch, false)
 				return
 			}
-
-			if priceData.Price.IsZero() || priceData.Price.IsNegative() {
-				s.logger.WithFields(log.Fields{
-					"ticker":   priceData.Ticker,
-					"provider": priceData.ProviderName,
-				}).Errorln("got negative or zero price, skipping")
-				continue
+			if priceData.OracleType != oracletypes.OracleType_Stork {
+				if priceData.Price.IsZero() || priceData.Price.IsNegative() {
+					s.logger.WithFields(log.Fields{
+						"ticker":   priceData.Ticker,
+						"provider": priceData.ProviderName,
+					}).Errorln("got negative or zero price, skipping")
+					continue
+				}
+			} else {
+				if len(priceData.AssetPairs) == 0 {
+					s.logger.WithFields(log.Fields{
+						"ticker":   priceData.Ticker,
+						"provider": priceData.ProviderName,
+					}).Errorln("got zero asset pair for stork oracle, skipping")
```

Comment:
this doesn't seems to be a real error, just something that can happen
```suggestion
					}).Debugln("got zero asset pair for stork oracle, skipping")
```
we can avoid this check here completely if we do not push a `priceData` without any pair as explained in the comment in `processSetPriceFeed` function